.github/workflows/main.yml

```diff
@@ -2,7 +2,7 @@ name: Deploy Hugo site to Pages
 
 on:
   push:
-    branches: ["main", "student-talk"]
+    branches: ["main", "software-entries"]
```

Comment:
Same question [as the other PR](https://github.com/open-neuromorphic/open-neuromorphic.github.io/pull/285#discussion_r2202907725): do we want to publish the docs when the branch gets pushed?